bin/sozo/src/commands/call.rs

```diff
@@ -0,0 +1,63 @@
+use anyhow::Result;
+use clap::Args;
+use scarb::core::Config;
+use starknet::core::types::FieldElement;
+
+use super::options::account::AccountOptions;
+use super::options::starknet::StarknetOptions;
+use super::options::world::WorldOptions;
+use crate::utils;
+
+#[derive(Debug, Args)]
+#[command(about = "Call a system with the given calldata.")]
+pub struct CallArgs {
+    #[arg(help = "The address or the fully qualified name of the contract to call.")]
+    pub contract: String,
+
+    #[arg(help = "The name of the entrypoint to call.")]
+    pub entrypoint: String,
+
+    #[arg(short, long)]
+    #[arg(value_delimiter = ',')]
+    #[arg(help = "The calldata to be passed to the entrypoint. Comma separated values e.g., \
+                  0x12345,0x69420.")]
+    pub calldata: Vec<FieldElement>,
+
+    #[arg(short, long)]
+    #[arg(help = "The block ID (could be a hash, a number, 'pending' or 'latest')")]
+    pub block_id: Option<String>,
+
+    #[command(flatten)]
+    pub starknet: StarknetOptions,
+
+    #[command(flatten)]
+    pub account: AccountOptions,
```

Comment:
```suggestion
```
Account should not be required to make a call. But it seems the utils function requires it. We should do an other utils function without account (using the `WorldContractReader` instead, as it a call should remain very light weight.

So one want to do:
```
sozo call my::contract::actions tile -c 1,2 --world 0x1234
```
without actually pointing to any `Scarb.toml` to grasp the configuration.

This changes impact:
1. `bin/sozo/utils` -> a new function to init the `WorldContractReader` from the env metadata.
2. `sozo/ops/lib.rs` -> a new function to get the contract address from the reader.
3. Changes related to that into the commands and ops for `call.rs`.

Reply:
Yes, good idea ! Thanks !
I've updated the PR and moved the `parse_block_id` function to the `utils` crate as it might be useful for another command.